packages/fxa-auth-server/lib/senders/emails/partials/location/index.mjml

```diff
@@ -1,30 +1,27 @@
-<%# This Source Code Form is subject to the terms of the Mozilla Public
-  # License, v. 2.0. If a copy of the MPL was not distributed with this
-  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+<%# This Source Code Form is subject to the terms of the Mozilla Public #
+License, v. 2.0. If a copy of the MPL was not distributed with this # file, You
+can obtain one at http://mozilla.org/MPL/2.0/. %>
 
 <mj-include path="./css/location/index.css" type="css" css-inline="inline" />
 
 <mj-section>
   <mj-column>
-    <mj-text css-class="text-body-grey">
-    <% if (locals.primaryEmail) { %>
+    <mj-text css-class="<%= locals.cssClass || 'text-body-grey' %>">
+      <% if (locals.primaryEmail) { %>
```

Comment:
Eek, I don't like that prettier is doing this.

Reply:
The extra tab? That one actually seems correct to me since it's in `mj-text` but yeah I'll revert some of these prettier changes.

Reply:
Looks like the resulting formatting is flat:

<img width="225" alt="Screen Shot 2021-09-10 at 11 40 27 AM" src="https://user-images.githubusercontent.com/6392049/132871467-2c563fa4-a2cc-48e1-9de4-1781c288e09a.png">